docs/DeveloperGuide.adoc

```diff
@@ -4,7 +4,7 @@
 :toc-title:
 :toc-placement: preamble
 :sectnums:
-:imagesDir: images
+:imagesDVir: images
```

Comment:
Caused a bug that prevented images from showing in DG. WIll create a new commit to solve issue.